### PR TITLE
Improvements to install_stack.sh

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@gsong What do you think?

**Comment by feanil** [comment]
:+1:

File: util/install/install_stack.sh [hunk 1]
```diff
@@ -106,6 +115,7 @@ exec > >(tee install-$(date +%Y%m%d-%H%M%S).log) 2>&1
 echo "Capturing output to install-$(date +%Y%m%d-%H%M%S).log."
 
 export OPENEDX_RELEASE=$release
+echo "Installing release '$OPENEDX_RELEASE'"
```
Comment (gsong) [inline util/install/install_stack.sh:118]: I think `${ENV_VAR}` is the preferred interpolation these days.

File: util/install/install_stack.sh [hunk 2]
```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-# Stop if any command fails
+# Setting OPENEDX_DEBUG makes this more verbose.
+if [[ ! -z "$OPENEDX_DEBUG" ]]; then
```
Comment (gsong) [inline util/install/install_stack.sh]: Would `[[ $OPENEDX_DEBUG ]]` be sufficient?

File: util/vagrant/upgrade.sh [hunk 3]
```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-# Stop if any command fails
+# Setting OPENEDX_DEBUG makes this more verbose.
+if [[ ! -z "$OPENEDX_DEBUG" ]]; then
```
Comment (gsong) [inline util/vagrant/upgrade.sh]: Same here, would `[[ $OPENEDX_DEBUG ]]` be sufficient?

File: util/install/install_stack.sh (hunk 1 above)
Comment (nedbat) [reply util/install/install_stack.sh:118]: I'm told there is no difference between the two if there are no options being used, and no ambiguity about the ends of the name.

Comment (gsong) [reply util/install/install_stack.sh:118]: Functionally there's no difference. I just like to be consistent.

File: util/install/install_stack.sh (hunk 2 above)
Comment (nedbat) [reply util/install/install_stack.sh]: OK, made this change.

File: util/install/install_stack.sh (hunk 1 above)
Comment (nedbat) [reply util/install/install_stack.sh:118]: I usually try for the more noise-free option.

**Comment by gsong** [comment]
👌

**Pull request merged**